source/threads.tex

```diff
@@ -2968,10 +2968,8 @@
 \begin{itemdescr}
 \pnum
 \expects
-The \tcode{order} argument is neither
-\tcode{memory_order::consume},
-\tcode{memory_order::acquire}, nor
-\tcode{memory_order::acq_rel}.
+The \tcode{order} argument is either \tcode{memory_order::relaxed},
```

Comment:
Strictly speaking, "order" is not an argument, but the name of a parameter.

Reply:
So do you wish me to replace "argument" by "parameter"?

Reply:
I decided to perform this replacement